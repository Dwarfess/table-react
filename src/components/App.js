import React, { Component } from 'react';
import { BrowserRouter as Router, Route} from "react-router-dom";

import TaskList from '../containers/task-list';
import TaskInfo from '../containers/task-info';
import './App.css';

class App extends Component {
    render() {
        return (
            <Router>
                <div className="ui container">
                    <h1 className="App-title">Welcome to React</h1>

                    <Route path='/' exact component={TaskList} />
                    <Route path='/:task' exact component={TaskInfo} />
                </div>
            </Router>
        );
    }
}

export default App;
