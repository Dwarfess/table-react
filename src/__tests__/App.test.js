import React from 'react';
import App from '../components/App';
import {shallow} from 'enzyme';

describe('testing App component', () => {
  let wrapper;
  beforeEach(() => {wrapper = shallow(<App />)});

  it('includes 1 div with class container', () => {
    // expect(wrapper).toMatchSnapshot();
    expect(wrapper.find('div')).toEqual(1);
  });
});
