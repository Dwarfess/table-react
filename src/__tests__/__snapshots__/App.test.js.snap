// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `undefined`;

exports[`testing App component includes 1 div with class container 1`] = `undefined`;
